### ISSUE TITLE
Issue with Exercise 1

### ISSUE THREAD

**Issue by jackson-dean** [ember-best-practices/memory-leak-examples#17]
Following along with exercise 1, when traversing the snapshot retainers, there is not an obvious `_data` property to be found which could be used to discover the problem. I'm not sure if the issue is because of changes to Chrome or changes to Ember, but the only way to find the source of the memory leak is to look at the module's application code directly. Although the container is retained in memory, there is no way to discover the source from what is presented in the heap snapshot.

**Comment by trentmwillis** [comment]
Hmm, did you install dependencies using `yarn`? In general, the code for the exercises shouldn't have changed which likely means that Chrome may have changed and we'll need to update the guides.

**Comment by jackson-dean** [comment]
Yep, I ran `yarn install`. I believe the change is in chrome. Something to do with how weak maps are retained and presented in the heap snapshot. It has completely obfuscated the source of the leak to the point that it is unusable even for the simplest of examples.   
https://bugs.chromium.org/p/chromium/issues/detail?id=827718

**Comment by crixx** [comment]
I agree. With the current version of Chrome (Version 69.0.3497.100) it's impossible to find memory leaks as explained in the guide. The guide was our last hope to find the reason for the memory leaks which are blowing up our build after having switched to the new ember test-style... and there seems to be no other source of information about finding memory leaks in ember apart from this guide 😞 

Any hints or ideas?
